src/python/CRABClient/Commands/status.py

```diff
@@ -908,8 +908,10 @@ def printPublication(self, publicationEnabled, jobsPerStatus, numProbes, numUnpu
             self.logger.info(msg)
             return pubStatus
         ## Case in which there was an error in retrieving the publication status.
+        pubSource = "\n(from Oracle transfer DB)"
```

Comment:
let's put something whjich makes sense to the user, and yet is useful to us,  like "from internal CRAB transferdb"

Reply:
better: "from internal CRAB bookkeeping in transferdb"